deployment/README.md

```diff
@@ -0,0 +1,44 @@
+# Deployment
+
+## AWS CDK Account Bootstrap Instructions
+
+Before deploying this AWS CDK project into an AWS account, the account must be bootstrapped. Bootstrapping is required to set up the necessary resources (like an S3 bucket and IAM roles) that CDK uses internally for deployments.
+
+Due to restrictions in our AWS accounts, the standard CDK bootstrap template cannot be used. Instead, we have a custom bootstrap template (`bootstrap.yml`) stored in this repository.
+
+### Bootstrapping Instructions
+
+To bootstrap an AWS account using our custom template, follow these steps:
+
+1. Create a temporary CDK app directory:
+
+   ```bash
+   mkdir hello-cdk
+   cd hello-cdk
+   ```
+
+2. Initialize a new CDK app:
+
+   ```bash
+   cdk init app --language typescript
+   ```
+
+3. Bootstrap the AWS account using the custom bootstrap template:
+
+   ```bash
+   cdk bootstrap aws://<account number>/us-east-1 --template ../bootstrap.yml
```

Comment:
```suggestion
   curl https://raw.githubusercontent.com/aws/aws-cdk/refs/heads/main/packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml > bootstrap.yaml
   cdk bootstrap aws://<account number>/us-east-1 --template ../bootstrap.yml
```
How would you feel about including this in the instructions so that we don't have to bring that boilerplate yaml in to the repo?

Reply:
We're not using that template exactly, we need to make some additions to it for CMS's accounts.  I'll create another PR to sync our template to it more closely.